test/test_common/network_utility.cc

```diff
@@ -166,14 +169,19 @@ std::pair<Address::InstanceConstSharedPtr, int> bindFreeLoopbackPort(Address::Ip
   Address::InstanceConstSharedPtr addr = getCanonicalLoopbackAddress(version);
   const char* failing_fn = nullptr;
   const int fd = addr->socket(type);
+  int err;
   if (fd < 0) {
+    err = errno;
```

Comment:
I think it'd be nice to have socket return a tuple for consistency.

Reply:
Will do.

Reply:
Addressed in 6c65b13.